Update const-fn doc in unstable-book (#79858)
rust-lang/rust: src/doc/unstable-book/src/language-features/const-fn.md

------------------------

> The `const_fn` allows to enable additional functionality
> to already stabilized [minimal subset of `const_fn`](https://github.com/rust-lang/rust/issues/53555)

Reviewer: ```suggestion
`const_fn` feature enables additional functionality not stabilized in the
[minimal subset of `const_fn`](https://github.com/rust-lang/rust/issues/53555)
```
Would this be clearer?

Reviewer 2: I think we should keep the `The` in front, but otherwise, yes.

Reviewer: I have that originally but I removed it.
```suggestion
The `const_fn` feature enables additional functionality not stabilized in the
[minimal subset of `const_fn`](https://github.com/rust-lang/rust/issues/53555)
```
